exe/node/Main.hs

```diff
@@ -4,7 +4,7 @@ import           Oscoin.Prelude hiding (option)
 
 import qualified Oscoin.API.HTTP as HTTP
 import           Oscoin.CLI.KeyStore (readKeyPair)
-import           Oscoin.CLI.Parser (environmentParser, keyPathParser)
+import           Oscoin.Configuration
```

Comment:
Drat, I have introduced the same module in my address-format PR -- one of us will get conflicts 😁